hack/test-cmd.sh

```diff
@@ -576,7 +576,7 @@ oc create -f test/integration/fixtures/test-buildcli.json
 # a build for which there is not an upstream tag in the corresponding imagerepo, so
 # the build should use the image field as defined in the buildconfig
 started=$(oc start-build ruby-sample-build-invalidtag)
-oc describe build ${started} | grep openshift/ruby-20-centos7$
+oc describe build ${started} | grep atomicenterprise/ruby-20-centos7$
```

Comment:
This was to be part of the follow on work but it's been split. Will have a follow on PR.